kolibri/core/content/api.py

```diff
@@ -291,6 +294,14 @@ def filter_available(self, queryset, name, value):
         return queryset.filter(root__available=value)
 
 
+class ChannelThumbnailView(View):
```

Comment:
For writing unit test for this is there any existing `setUp` method that will set up a channel metadata model?

Reply:
If you look in the `test_content_app.py` file in the `content` app, you should see some examples.